src/plugins/ui_core_views/evaluation_data_validation.ts

```diff
@@ -228,9 +226,11 @@ export class EvaluationDataValidationPlugin extends UIPlugin {
         );
 
         const evaluated = this.getters.evaluateFormula(sheetId, translatedFormula);
-        return evaluated && !isMatrix(evaluated) ? evaluated.toString() : "";
+        return evaluated?.toString() || "";
```

Comment:
what if it's a matrix? Is it safe to remove the check?

Reply:
I don't know what we want exactly:

Here I removed the check because the try catch below takes care of this case.
The `evaluateFormula` getter has been modified to throw when an array is received.
Perhaps we should first ask ourselves what exactly we want to do with this getters when it returns an array ?

Reply:
Ok understood. After live discussion: `this.getters.evaluateFormula` shouldn't throw